css-brochure-form/index.html

```diff
@@ -13,27 +13,31 @@
     <div id="h1"><h1>LEARN MORE</h1></div>
 
     <form action="#">
-      <label for="companyEmail">COMPANY EMAIL</label><br />
-      <input type="text" placeholder="sally@company.co" id="companyEmail" />
-      <br />
+      <div>
+        <label for="companyEmail">COMPANY EMAIL</label>
+        <input type="text" placeholder="sally@company.co" id="companyEmail" />
+      </div>
 
-      <label for="name">NAME</label><br />
-      <input type="text" placeholder="Sally Smith" id="name" /><br />
+      <div>
+        <label for="name">NAME</label>
+        <input type="text" placeholder="Sally Smith" id="name" />
+      </div>
 
-      <label for="hiringLocation">WHERE ARE YOU HIRING?</label><br />
-      <input
-        type="text"
-        placeholder="City, Country"
-        id="hiringLocation" /><br />
+      <div>
+        <label for="hiringLocation">WHERE ARE YOU HIRING?</label>
+        <input type="text" placeholder="City, Country" id="hiringLocation" />
+      </div>
 
-      <label for="engineersNumber"
-        >HOW MANY ENGINEER ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
-      ><br />
-      <select name="engineersNumber" id="engineersNumber">
-        <option value="Opt1">1-2 Engineers</option>
-        <option value="Opt2">3-10 Engineers</option>
-        <option value="Opt2">More than 10 Engineers</option></select
-      ><br />
+      <div>
+        <label for="engineersNumber"
+          >HOW MANY ENGINEER ARE YOU HIRING IN THE NEXT 6 MONTHS?</label
+        ><br />
```

Comment:
You still have 1 `<br />` here

Reply:
My bad. Now I am annoyed with <br/> and dont want to see it in my code again :P